pkgs/development/python-modules/netbox-qrcode/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, pillow
+, qrcode
+}:
+
+buildPythonPackage rec {
+  pname = "netbox-qrcode";
+  version = "0.0.11";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = "netbox-qrcode";
+    rev = "v${version}";
+    hash = "sha256-vED+B+GYIgThhyG3dS+rT68yRE4kiMaBOiB8jsUKLiY=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    qrcode
+    pillow
+  ];
+
+  # Can't use pythonImportsCheck since it calls some netbox specific modules
```

Comment:
This can be fixed by using something like this.

https://github.com/NixOS/nixpkgs/pull/307786/files#diff-072632e3ed06eb61f815f4a38e9b79d1b9ed364add98c18ac0ccb704f1a104ceR23-R29